src/mono/wasm/runtime/assets.ts

```diff
@@ -64,6 +98,7 @@ export async function mono_download_assets(): Promise<void> {
     runtimeHelpers.maxParallelDownloads = runtimeHelpers.config.maxParallelDownloads || runtimeHelpers.maxParallelDownloads;
     try {
         const promises_of_assets_with_buffer: Promise<AssetWithBuffer>[] = [];
+        const skipIcuByAssetNames = get_skipped_icu_assets();
```

Comment:
please move this outside of the function. we should not initialize the list on each call to `resolve_asset_path`